docs/administration/configuration/configuring-storage-volumes/index.md

```diff
@@ -0,0 +1,194 @@
+It is also possible to configure a second storage volume to be used by REANA's infrastructure components, such as the database and the message broker.
+
+## Configuration
+
+Volumes can be configured with two Helm values:
+
+- [`shared_storage`](https://github.com/reanahub/reana/tree/master/helm/reana) holds the configuration for the shared filesystem needed by REANA.
+  It is mandatory to configure this volume.
+- [`infrastructure_storage`](https://github.com/reanahub/reana/tree/master/helm/reana) configures the second optional volume used by the database and the message broker.
+  If not configured, the shared storage volume is used instead.
+  It has the same settings as [`shared_storage`](https://github.com/reanahub/reana/tree/master/helm/reana).
+
+### CephFS
+
+To use an existing CephFS share, set the [`*_storage.backend`](https://github.com/reanahub/reana/tree/master/helm/reana) Helm value to `cephfs`.
+You can then configure the CephFS share by setting the appropriate values in [`*_storage.cephfs`](https://github.com/reanahub/reana/tree/master/helm/reana).
+
+This is an example CephFS configuration, using the default values present in the Helm chart:
+
+```{ .yaml .copy-to-clipboard }
+shared_storage: # or `infrastructure_storage`
+  backend: cephfs
+  volume_size: 200
+  access_modes: ReadWriteMany
+  cephfs:
+    provisioner: manila-provisioner
+    type: "Geneva CephFS Testing"
+    availability_zone: nova
+    os_secret_name: os-trustee
+    os_secret_namespace: kube-system
+    cephfs_os_share_id: <cephfs-share-id>
+    cephfs_os_share_access_id: <cephfs-share-access-id>
+```
+
+For more information regarding CERN deployments, see the related [Kubernetes@CERN documentation page](https://kubernetes.docs.cern.ch/docs/storage/fileshares/#using-existing-shares).
+
+### NFS
+
+You can deploy an NFS file system inside your cluster using the official [NFS Server Provisioner](https://github.com/helm/charts/tree/master/stable/nfs-server-provisioner) Helm chart.
+
+1. Create your configuration file, `nfs-provisioner-values.yaml`, following the [documentation](https://github.com/helm/charts/tree/master/stable/nfs-server-provisioner#recommended-persistence-configuration-examples) and adapting it to your needs. Remember to name the storage class as REANA expects it, `<helm-release-prefix>-shared-volume-storage-class`.
+
+    ```yaml hl_lines="3"
+    storageClass:
+      defaultClass: true
+      name: reana-shared-volume-storage-class
+    ```
+
+2. Install the NFS Server Provisioner
+
+    ```{ .console .copy-to-clipboard }
+    $ helm install reana-dev-storage stable/nfs-server-provisioner \
+                  -f nfs-provisioner-values.yaml
+    ```
+
+3. Configure the NFS storage volume in your Helm configuration file
+
+    ```{ .yaml .copy-to-clipboard }
+    shared_storaged: # or `infrastructure_storage`
+      backend: nfs
+      volume_size: 200
+      access_modes: ReadWriteMany
+    ```
+
+### HostPath
+
+!!! warning
```

Comment:
Shall we put the warning _after_ the first paragraph explaining what host path is and what it is suitable for?

(IOW, invert order of 1st-2nd paragraph?)